shipitsteven/note-app
# Saving now saves to the selected notes

## Description of Issue
Saving now saves to the selected note
Updated function signature of `Save` Sameen wrote

## Testing
*Have you tested this change? On what devices?*
Windows - Electron

### To Test
- Run app
- Select a note
- Edit note
- Press save
- Select another note, edit and save
- Both notes should reflect their updated value, with changes specific to their own edit

## Time Spent
- *How long did you estimate this task would take?* 1 hour
- *How long did it take you to work on this issue?* 30 minutes

## Checklist
